### PR TITLE
Rips out military branch/rank system

### LINKED ISSUE #1318: Job branches should be removed
All foundation jobs should be organized based on the class system. For some reason, many use the default Bay military branch system. Not only is this immersion breaking, it's also bloat. The entire system should be removed.

### PR DESCRIPTION
**Pull request by aidan-bowman**
## About the Pull Request

Closes #1318.
Also deletes some extraneous, unused assets.

## Why It's Good For The Game

Fuck bloated shitcode

## Changelog

:cl:
add: All classes (security included) use SCP classes instead of bay ones.
del: Deletes military branches/ranks and replaces them with simpler classes.
del: Deleted some jobcode related to species.
del: Deletes various items that are not lore-relevant/which are unused.
refactor: All code relating to branches/ranks has been refactored.
spritedel: Deleted a few old, unused bay assets.
/:cl:

### CODE REVIEW

**Comment by TheDarkElites** [comment]
Finally

**Comment by tichys** [comment]
why do you hurt me so

**Comment by aidan-bowman** [comment]
code so bad even the Add PR Labels ain't let it thru

**Comment by aidan-bowman** [comment]
converting to draft until i can fix things from the merge conflict

File: code/game/jobs/job/job.dm [hunk 1]
```diff
@@ -2,69 +2,88 @@
...
-	/// The main department this job is in.
+	/// If you have use_age_restriction_for_jobs config option enabled and the database set up, this option will add a requirement for players to be at least minimal_player_age days old. (meaning they first signed in at least that many days before.)
+	var/minimal_player_age = 0
+	/// Does this position have a department tag? // TODO: move to department_flag
 	var/department = null
-	/// If true, this job is counted as a head of its department
-	var/head_position = FALSE
-	var/minimum_character_age				// List of species = age, if species is not here, it's auto-pass
+	/// Is this position a head position (i.e. part of Command)?
+	var/head_position = 0
```
Comment (TheDarkElites) [inline code/game/jobs/job/job.dm]: FALSE is just a define for 0, so changing this to zero just decreases readability.

File: code/modules/client/preference_setup/background/03_records.dm [hunk 2]
```diff
@@ -4,7 +4,6 @@
 	var/sec_record = ""
 	var/gen_record = ""
 	var/memory = ""
-	var/email_addr = ""
```
Comment (TheDarkElites) [inline code/modules/client/preference_setup/background/03_records.dm:7]: Why are we fucking with emails here?

File: code/modules/clothing/under/accessories/armband.dm [hunk 3]
```diff
@@ -36,11 +36,6 @@
 	desc = "An armband, worn by the crew to display which department they're assigned to. This one is white and green."
 	icon_state = "medgreen"
 
-/obj/item/clothing/accessory/armband/medblue
```
Comment (TheDarkElites) [inline code/modules/clothing/under/accessories/armband.dm]: Why?

File: code/modules/submaps/submap_job.dm [hunk 4]
```diff
@@ -45,18 +43,6 @@
 		owner = _owner
 		..()
 
-/datum/job/submap/is_species_allowed(datum/species/S)
```
Comment (TheDarkElites) [inline code/modules/submaps/submap_job.dm:48]: While we do not have different species, this seems out of scope and should be declared in the pr changelogs to bring it into scope.

Same for most of the clothing items that were removed.

File: maps/site53/items/uniform_vendor.dm
Comment (TheDarkElites) [inline maps/site53/items/uniform_vendor.dm:1]: This should be added to pr desc as its currently out of scope.

File: packs/faction_iccgn/badges.dm
Comment (TheDarkElites) [inline packs/faction_iccgn/badges.dm:1]: Also out of scope, add to PR desc.

**Review by TheDarkElites — CHANGES_REQUESTED** [review]
PR desc needs to be modified since there are quite a few out of scope changes. Dont remove the changes, just bring em' into scope by modifying the pr desc.

**Comment by TheDarkElites** [comment]
Also currently untested so make sure cheese tests it before its merged

File: code/game/jobs/job/job.dm (hunk 1 above)
Comment (aidan-bowman) [reply code/game/jobs/job/job.dm]: Fixed

File: code/modules/client/preference_setup/background/03_records.dm (hunk 2 above)
Comment (aidan-bowman) [reply code/modules/client/preference_setup/background/03_records.dm:7]: E-mail domains are set by branches... which were removed. Ergo, e-mail code was simplified. There was no way to have a domain other than `site53.foundation` anyways.

File: packs/faction_iccgn/badges.dm
Comment (aidan-bowman) [reply packs/faction_iccgn/badges.dm:1]: Included

File: maps/site53/items/uniform_vendor.dm
Comment (aidan-bowman) [reply maps/site53/items/uniform_vendor.dm:1]: Included

File: code/modules/submaps/submap_job.dm (hunk 4 above)
Comment (aidan-bowman) [reply code/modules/submaps/submap_job.dm:48]: Included

**Comment by aidan-bowman** [comment]
> Also currently untested so make sure cheese tests it before its merged

Alright tested it, there was a bug with IDs but I fixed it.

File: code/modules/client/preference_setup/background/03_records.dm (hunk 2 above)
Comment (TheDarkElites) [reply code/modules/client/preference_setup/background/03_records.dm:7]: aight its fine then

File: code/modules/clothing/under/accessories/armband.dm (hunk 3 above)
Comment (aidan-bowman) [reply code/modules/clothing/under/accessories/armband.dm]: fixed

**Review by TheDarkElites — APPROVED** [review]
Codewise looks fine
Test wise:
image
he said it wouldent break :)
